### PR TITLE
Belly Interaction Improvements

### PR DESCRIPTION
**Pull request by AmatsuDF**
Adds a secondary transfer location for vore bellies.
Also recolors certain belly modes for easier identification at a glance what mode a belly is in.

### CODE REVIEW

**Comment by Very-Soft** [comment]
would like to ensure this is well tested before I merge it

**Comment by AmatsuDF** [comment]
Understandable. I did some basic testing with an alt and everything seemed to function as intended, but a second set of eyes on things never hurts given it is some (light) UI edits to support it.

File: code/modules/vore/eating/belly_obj_vr.dm [hunk 1]
```diff
@@ -762,6 +766,27 @@
...
+			var/obj/belly/dest_belly
+			for(var/obj/belly/B as anything in owner.vore_organs)
+				if(B.name == transferlocation_secondary)
+					dest_belly = B
+					break
+
+			if(!dest_belly)
+				to_chat(owner, "<span class='warning'>Something went wrong with your belly transfer settings. Your <b>[lowertext(name)]</b> has had it's transfer chance and transfer location cleared as a precaution.</span>")
+				transferchance = 0
+				transferlocation = null
```
Comment (Arokha) [inline code/modules/vore/eating/belly_obj_vr.dm:779]: This is still setting the primary to nothing rather than the secondary. Probably copy-paste error.

**Review by Arokha — CHANGES_REQUESTED** [review]

File: code/modules/vore/eating/belly_obj_vr.dm (hunk 1 above)
Comment (AmatsuDF) [reply code/modules/vore/eating/belly_obj_vr.dm]: Oops. Thank you.

**Comment by AmatsuDF** [comment]
Trying to make the checks complete on the advice of those in the Discord.

**Pull request merged**